src/ko/newtoki/src/eu/kanade/tachiyomi/extension/ko/newtoki/NewToki.kt

```diff
@@ -183,7 +183,9 @@ abstract class NewToki(
         mangaDetailsParse(document).apply {
             // TODO: don't throw when there is download folder rename feature
             if (manga.description.isNullOrEmpty() && manga.title != title) {
```

Comment:
```suggestion
            if (manga.description.isNullOrEmpty() && title.removeSuffix("…") !in manga.title) {
```

I think you actually mean this. Could you provide a link to an entry with very long title?

Reply:
yes this code good